server/pulp/plugins/loader/manager.py

```diff
@@ -65,8 +63,6 @@ def _load_unit_models(self):
 
             # Attach all the signals
             model_class.attach_signals()
-            signals.post_init.connect(model_class.post_init_signal, sender=model_class)
```

Comment:
Does `TaskStatus` need to implement `attach_signals()`?  Here is how it currently does it:

https://github.com/pulp/pulp/blob/master/server/pulp/server/db/model/__init__.py#L314